plenoptic/simulate/models/naive.py

```diff
@@ -235,27 +237,28 @@ def __init__(
 
         self.cache_filt = cache_filt
         self._filt = None
+        self._input_device = torch.device("cpu")
```

Comment:
I think this is unused, yeah? in which case, should delete it

Reply:
done